test/functional/inspec_init_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'functional/helper'
+require 'fileutils'
+require 'tmpdir'
+require 'yaml'
+
+describe 'inspec init' do
+  include FunctionalHelper
+
+  tmpdir = Dir.tmpdir
+  puts tmpdir
+
+  describe 'inspec init profile with/slash' do
+    it 'Names profile with string after last slash' do
+      slash_profile = "#{tmpdir}/inspecwith/slash"
+      out = inspec("init profile #{slash_profile}")
+      out.exit_status.must_equal 0
+      File.exist?(slash_profile).must_equal true
+      profile = YAML.load_file("#{slash_profile}/inspec.yml")
+      profile["name"].must_equal 'slash'
```

Comment:
Single quote since no interpolation?

NOTE: rubcop doesn't cover tests so...I can't fault you if you leave it.